.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "third_party/nanopb"]
 	path = third_party/nanopb
 	url = https://github.com/nanopb/nanopb.git
+[submodule "tools/thrift"]
```

Comment:
I think we keep all external modules in third_party/ . Any reason to put this in tools/ ?